pkg/ibmvpc/sg_analysis.go

```diff
@@ -338,6 +338,21 @@ func mapAndAnalyzeSGRules(rules []*SGRule, isIngress bool, connectivityMap Conne
 	disjointLocals := ipblock.DisjointIPBlocks(locals, []*ipblock.IPBlock{ipblock.GetCidrAll()})
 	keysToConnectivityResult := map[common.SetAsKey]*ConnectivityResult{}
 	for i := range disjointLocals {
+		localContainsMember := false
+		for member := range currentSg.members {
+			memberIPBlock, err := ipblock.FromIPAddress(member)
+			if err != nil {
+				return
+			}
+			if memberIPBlock.ContainedIn(disjointLocals[i]) {
+				localContainsMember = true
+				break
```

Comment:
can add a function (method for SecurityGroup) that returns an `*IPBlock` object with union of all members, have this value computed once outside the loop, and for each `disjointLocal` check that it has non-empty intersection with this value.

Reply:
I used this specific way because it can be non-empty intersection with this value but still we do not need this disjointLocal (no member alone contained in this disjointLocal)

Reply:
I meant that each `disjointLocal` has to be checked with non empty intersection of the unified members ipBlock.
this means that at least one member is contained in this  `disjointLocal`.